packages/charts/src/core/chart-assists/types.ts

```diff
@@ -67,6 +67,5 @@ export class ChartAssistRenderStateData implements IRenderStateData {
     }
 }
 
-export interface IRenderStatesIndex {
-    [seriesId: string]: ChartAssistRenderStateData;
```

Comment:
Does changing this remove some intellisense with the self-documenting `seriesId` key description? With Record you can't tell exactly what the keys are supposed to be.